libs/libcommon/src/libcommon/exceptions.py

```diff
@@ -463,13 +463,6 @@ def __init__(self, message: str, cause: Optional[BaseException] = None):
         )
 
 
-class SplitWithTooBigParquetError(CacheableError):
```

Comment:
To automatically recompute the stats for the big splits, we could add `SplitWithTooBigParquetError` to the list of retryable errors, and the first backfill cron job run after the deployment should create the missing jobs.

And later, once the database is free from this error code, we could remove it.

Reply:
I think we should do it. See https://github.com/huggingface/datasets-server/blob/a5c69da73ba8fe9e419e6ca1996134ab645ccf42/libs/libcommon/src/libcommon/constants.py#L38-L45

We can add the error code there, with a comment to remove it later